department-of-veterans-affairs/content-build
# Add registry for 21-4142 medical release form

## Description

closes https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/88
relates to 
- department-of-veterans-affairs/va.gov-team-forms#88

## Testing done & Screenshots
### `http://localhost:3001/health-care/medical-release-form/introduction`

![image](https://user-images.githubusercontent.com/123402053/223228159-99d65aa6-76de-4709-a1c0-bdd7d92faee5.png)

## QA steps

What needs to be checked to prove this works? 
- Once this PR and the vets-website PR is merged, should be able to visit `http://localhost:3001/health-care/medical-release-form/introduction` and see the new app.  
What needs to be checked to prove it didn't break any related things?  
- This is a new app and `"vagovprod": false `


## Acceptance criteria
- [x] The new route is accessible
